app/src/main/java/de/tum/in/tumcampusapp/api/app/TUMCabeClient.java

```diff
@@ -234,7 +236,7 @@ public void deviceUploadGcmToken(DeviceUploadGcmToken verification, Callback<TUM
                .enqueue(cb);
     }
 
-    public void createMeasurements(List<WifiMeasurement> wifiMeasurementList, Callback<TUMCabeStatus> cb) throws IOException {
```

Comment:
@kordianbruck there are still errors in RoomFinderDetailsActivity, because that IOException isn't declared anymore...

Reply:
Gradle incremental builds sometimes aren't fun... Thanks for catching this.